### PR TITLE
Task requirements

### PR DESCRIPTION
**Pull request by NishSatish**
A Google Code-in 2017 task.

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -1 +1,3 @@
-1. Dilpreet Singh (http://github.com/dilpreet96) : Open source contributor
```
Comment (AbhilashG001) [inline Contributors.md:1]: Hello @NishSatish,

You don't have to remove Dilpreet's name form the list, rather you only have to add your name on the list. Can you please revert this change, and add your name after Dilpreet's name ?

After you have made this change, please use ```git commit --amend``` and then **force** push your changes to send a pull request. You can also [squash](https://stackoverflow.com/questions/2563632/how-can-i-merge-two-commits-into-one) your commits after making the suggested change if you are comfortable with git. 

Please let us know if you need any help, we'll be glad to assist you.

Comment (NishSatish) [reply Contributors.md]: I have completed the required changes. I then opened a pull request. I have commited the file again, please review it Sir @AbhilashG97 .

**Review by NishSatish — COMMENTED** [review]
Required changes completed.

File: Contributors.md (hunk 1 above)
Comment (NishSatish) [reply Contributors.md]: Sir @AbhilashG97 can I restart the task?

**Comment by NishSatish** [comment]
Dear sir, i have made the requested changes. Since I encountered terrible
error (Travis CI Build error) during the second commit, I had to fork the
repository again and make a new pull request. But it is done now, and
submitted for review.

But may I know what the 'Travis CI build system' is? It maybe helpful in
the future.

Nishant S

**Pull request closed without merging**